backend/handlers/users/credit/usercredit_test.go

```diff
@@ -0,0 +1,68 @@
+package usercredit_test
```

Comment:
I don't think this should be a separate package. We're not exporting test functionality here.